SLHCUpgradeSimulations/L1TrackTrigger/plugins/L1TkMuonFromExtendedProducer.cc

```diff
@@ -66,14 +71,58 @@ class L1TkMuonFromExtendedProducer : public edm::EDProducer {
 
   };
 
+  struct L1MuonCand {
+    float eta;
+    float phi;
+    float pt;
+    float feta;
+    float bx;
+    float sigmaEta;
+    float sigmaPhi;
+    unsigned int quality;
+    int  idx;
+    bool gmtCand;
+    bool dtCand;
+    bool cscCand;
+    bool rpcCand;
+    int  itk;
+    int  Num;
+  };  
+
+  struct L1Tracks {
+    float pt;
+    int   nPars;
+    int   q;
+    float z;
+    float chi2;
+    float eta;
+    float phi;
+    int   nstubs;
+    int   idx;
+    float matchPt;
```

Comment:
this is an input L1 tracker track, it can't really have a "match" variable, it implies some logic from the algorithm.
We really need just to convert a C++ L1TTTrack to a list of variables that we need.
For now, just rename to propPt, propEta, propPhi ... (no other changes needed)